### PR TITLE
[Doc] Fix ConvexShapeContact documentation.

### PR DESCRIPTION
**Pull request by florent-lamiraux**
@jmirabel : can you double check ?

### CODE REVIEW

File: include/hpp/constraints/convex-shape-contact.hh [hunk 1]
```diff
@@ -35,24 +35,24 @@ namespace hpp {
...
                d_{\perp}^2                   &, \text{ otherwise}
              \end{array} \right.
         \f}
         where
-        \li \f$P (C_{o_i}, f_j)\f$ is the projection of the center \f$o_i\f$ onto the plane containing \f$ f_j \f$,
+        \li \f$P (C_{o_i}, f_j)\f$ is the projection of the center \f$C_{o_i}\f$ of \f$o_i\f$ onto the plane containing \f$ f_j \f$,
         \li \f$\textbf{n}_{f_j}\f$ is the normal of \f$ f_j \f$,
-        \li \f$d_{\parallel} = d(f_j, P (C_{o_i}, f_j))\f$ is the distance returned by ConvexShape::distance,
-        \li \f$d_{\perp} = \textbf{n}_{f_j}.C_{f_j}P(C_{o_i}, f_j)\f$ is the distance along the normal of \f$ f_j \f$,
+        \li \f$d_{\parallel} = d(C_{f_j}, P (C_{o_i}, f_j))\f$ is the distance returned by ConvexShape::distance,
```
Comment (jmirabel) [inline include/hpp/constraints/convex-shape-contact.hh:54]: This is the only place where I am not sure. I have to check the source code.

Comment (jmirabel) [reply include/hpp/constraints/convex-shape-contact.hh:54]: After double checking, this is not correct. Here we truly compute the distance between a point and a convex shape.

Comment (jmirabel) [reply include/hpp/constraints/convex-shape-contact.hh:54]: While here, could you:
- here, replace here the pointer to ConvexShape::distance to ConvexShapeData::distance
- in the doc of ConvexShapeData::distance, replace the pointer to ConvexShape::distance by ConvexShape::distanceLocal ?

Comment (florent-lamiraux) [reply include/hpp/constraints/convex-shape-contact.hh:54]: Yes, I realize that I introduced a mistake. I will fix it.

Comment (florent-lamiraux) [reply include/hpp/constraints/convex-shape-contact.hh]: I fixed the formula and updated the links as you requested.
I have also added a figure.

Comment (jmirabel) [reply include/hpp/constraints/convex-shape-contact.hh]: Thanks !

**Pull request merged**